lib/src/arithmetic/fixed_to_float.dart

```diff
@@ -107,6 +146,9 @@ class FixedToFloat extends Module {
                 .named('eShift'),
             Const(0, width: iWidth))
         .named('eRaw');
+
+    // TODO(desmonddak): this should be mantissa, but we could figure it out
```

Comment:
is this a functional issue or an optimization?

Reply:
Optimization.  or'ing the rounded mantissa is happening quite late in the computation.  We can likely get an earlier indication like and'ing the absolute value, but it needs to be exhaustively proven.

Reply:
I modified this comment and discovered that Dart linter is happy if I indent after I start a flutter TODO.  It keeps it as one single TODO instead of a first line and then a bunch of comments.